styleguide/derek/templates/thought-leadership-overview/_partials/related-topics.ejs

```diff
@@ -0,0 +1,23 @@
+<h2 class="rsTl-relatedTopics-header">Related Topics</h2>
+<div class="rsTl-relatedTopics-row">
+  <div class="rsTl-relatedTopics-col">
+    <div class="rsTl-relatedTopics-feature">
+      <div class="rsTl-relatedTopics-img" style="background-image:url('imgs/derek/doge-small.png');"></div>
```

Comment:
We may need a default image here as well.  Again, a question for @mikerastiello.